package.json

```diff
@@ -35,5 +35,10 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-standard": "^4.1.0",
     "nodemon": "^2.0.6"
+  },
+  "dependencies": {
+    "pg": "^8.5.1",
+    "pg-hstore": "^2.3.3",
+    "sequelize": "^6.3.5"
```

Comment:
Hey @LouangeMu you need also to install `sequelize-cli` package because you will need it when you will be creating models, seeds, run migrations etc